### PR TITLE
Add Timon/Titus

### PR DESCRIPTION
**Pull request by GingertronMk1**
Technical Operators needs to be checked and made right, I'm sure I've got a couple wrong

### CODE REVIEW

**Comment by BenWoodford76** [comment]
@GingertronMk1 Will check in a bit. Will hold off merging till tonight

File: _shows/17_18/timtit.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+- role: Poster and Publicity Manager
+  name: Lewis Scott
+- role: Shadow Technical Director
+  name: Laura Wolczyk
+- role: Shadow Sound Designer
+  name: Felicity Chilver
+- role: Shadow Sound Designer
+  name: Rohanna Brown
+- role: Technical Operator
+  name: Sam Osborne
```
Comment (samozzy) [inline _shows/17_18/timtit.md]: Add Hannah Burne, replace Maddy with Berrington.

Comment (GingertronMk1) [reply _shows/17_18/timtit.md]: I knew Bez replaced someone, couldn't remember who

**Comment by nhyty** [comment]
add prod shots when they are ready

**Pull request merged**